css/global.css

```diff
@@ -605,6 +639,28 @@ header nav li:last-child {
 Global Header Styles
 ===================================
 */
+
+header {
+	min-height: 100vh;
+	display: flex;
+	flex-direction: column;
+}
+
+/* Hero Image Component */
+.hero {
+	flex-grow: 1;
+	height: inherit;
+	background-repeat: no-repeat;
+	background-size: cover;
+	background-position: top center;
+	padding: 140px 0;
+	color: var(--color-white);
+}
+
+.hero-img-mobile {
```

Comment:
mobile will have an img hidden on page load. In mobile mq, will make visible and display block

Reply:
This is a great utility class that we can actually use with images throughout the site. I would rename it to `.img-mobile` and apply it to images outside of the hero as needed.